domains/wacky.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "A website for a developer's profile.",
+
+    "domain": "is-cool.dev",
+    "subdomain": "wacky",
+
+    "owner": {
+        "repo": "https://github.com/Arzejames/WackyDevWebsite",
+        "email": "turtles127855@gmail.com"
+    },
+
+    "record": {
+        "CNAME": ["http://arzejames.github.io/"]
```

Comment:
Invalid CNAME. Should be just a string no array, and should be a hostname with not protocols or directories.